bin/yb-ctl

```diff
@@ -1260,8 +1303,11 @@ class ClusterControl:
 
 if __name__ == "__main__":
     logging.basicConfig(
+            filename=SCRIPT_DEBUG_FILE,
             level=logging.INFO,
             format="%(asctime)s %(levelname)s: %(message)s")
+    print("Logging debug information to: {}".format(
+        os.path.join(os.getcwd(), SCRIPT_DEBUG_FILE)))
```

Comment:
I am not so sure about creating an odd `startup.log` file in the current directory. Also the output of every invocation of `yb-ctl` will now be cluttered with the "Logging debug information" message. Perhaps let's disable debug logging by default and still print it to stderr if a `--verbose` (`-v`) flag is specified?

Reply:
are you suggesting not logging anything anywhere if not `verbose`? just doing prints to the console for things like status?

Reply:
Yes, that's an option. In my opinion, command-line tools for interacting with servers should not produce a log file in the current directory unless explicitly asked to.

Reply:
I'm still not sure which way to go on this. I definitely do not want this to not log anything by default! It will clearly fail in various weird ways and if there's no log neither users nor us will be able to know what happened...

However, I also agree that dumping a log file in wherever we called the script from, is not ideal...The best workaround for that I think would be to drop it into the cluster data dir, but there's at least 2 issues with that that I can think of:
- we have to carefully massage when we setup when we parse the args (so we know what dir we need) vs when we setup the logging (so we can actually respect logging.info calls afterwards) vs when we do the rest of the code, including the one that actually creates the directory in which we want to log...
- if users run commands with and without specifying the data_dir, we'll end up sprawling some log files in various locations, potentially even causing inadvertent failures (say we create the default dir and put our log file there, then they stop using a custom dir and try the usual one, but it already exists, with only this log file there..)

Reply:
Discussed offline, added mechanics to dump to a temp file, clean it up if we did not have errors, else dump it to stderr if we did have errors. Also added `--verbose` to always dump logging to stderr regardless of success/fail